crates/next-api/src/module_graph.rs

```diff
@@ -740,15 +747,27 @@ impl ReducedGraphs {
     }
 
     #[turbo_tasks::function]
+    /// Validates that the global CSS/SCSS/SASS imports are only valid imports with the following
+    /// rules:
+    /// * The import is made from a `node_modules` package
+    /// * The import is made from a `.module.css` file
+    /// * The import is made from the `pages/_app.js`, or equivalent file.
     pub async fn validate_pages_css_imports(
         &self,
         entry: Vc<Box<dyn Module>>,
         app_module: Vc<Box<dyn Module>>,
     ) -> Result<()> {
         let span = tracing::info_span!("validate pages css imports");
         async move {
+            let graphs = &self.bare_graphs.await?.graphs;
+
+            // We need to collect the module names here to pass into the
+            // `validate_pages_css_imports` function. This is because the function is
+            // called for each graph, and we need to know the module names of the parent
+            // modules to determine if the import is valid. We can't do this in the
+            // called function because it's within a closure that can't resolve turbo tasks.
             let mut ident_vec = Vec::new();
```

Comment:
nit:  it would be slightly more efficient to do this as an `iter().map(...).try_join()` instead of a loop since it would allow us to execute each iteration 'concurrently'